bentoml/yatai/python_api.py

```diff
@@ -79,21 +87,59 @@ def upload_bento_service(bento_service, base_path=None, version=None):
         # Saving directory to path managed by LocalBentoRepository
         save_to_dir(bento_service, response.uri.uri)
 
-        upload_status = UploadStatus(status=UploadStatus.DONE)
-        upload_status.updated_at.GetCurrentTime()
-        update_bento_req = UpdateBentoRequest(
-            bento_name=bento_service.name,
-            bento_version=bento_service.version,
-            upload_status=upload_status,
-            service_metadata=bento_service._get_bento_service_metadata_pb(),
-        )
-        yatai.UpdateBento(update_bento_req)
+        update_yatai_status(yatai, bento_service)
 
         # Return URI to saved bento in repository storage
         return response.uri.uri
+    elif response.uri.type == BentoUri.S3:
+        with tempfile.TemporaryDirectory() as tmpdir:
+
+            save_to_dir(bento_service, tmpdir)
+
+            fileobj = io.BytesIO()
+            with tarfile.open(mode="w:gz", fileobj=fileobj) as tar:
```

Comment:
What's your plan on supporting creating BentoML bundle in tarball format and making that default?

Reply:
Right now only one format the tarball format, so it's the default.

Reply:
That's unexpected, I will look into it a bit more